docs/content/development/extensions-core/lookups-cached-global.md

```diff
@@ -147,13 +147,14 @@ setting namespaces (broker, peon, historical)
 |--------|-----------|-------|
 |`druid.lookup.namespace.cache.type`|Specifies the type of caching to be used by the namespaces. May be one of [`offHeap`, `onHeap`]. `offHeap` uses a temporary file for off-heap storage of the namespace (memory mapped files). `onHeap` stores all cache on the heap in standard java map types.|`onHeap`|
 |`druid.lookup.namespace.numExtractionThreads`|The number of threads in the thread pool dedicated for lookup extraction and updates. This number may need to be scaled up, if you have a lot of lookups and they take long time to extract, to avoid timeouts.|2|
+|`druid.lookup.namespace.numBufferedEntries`|If using offHeap caching, the number of records to be stored on an on-heap buffer|100,000|
```

Comment:
Other descriptions here end with a dot.

Also, could you highlight somehow (it may require to restructure somehow this part of the doc, including the following lines until the beginning of the next section) that this configuration must always be provided along with `offHeap`, because the size of each lookup is different and most likely the default value makes very little sense with the user's lookups?

Reply:
Fixed & attempted adding a line, thanks!.